packages/hash/frontend/src/blocks/page/BlockContextMenu/LoadEntityMenuContent.tsx

```diff
@@ -31,83 +28,50 @@ export const LoadEntityMenuContent: VFC<LoadEntityMenuContentProps> = ({
   popupState,
 }) => {
   const { accountId } = useRouteAccountInfo();
-  const { data: entities, fetchEntities, loading } = useAccountEntities();
   const blockView = useBlockView();
+  // This depends on state external to react without subscribing to it
+  // and this can cause some bugs.
+  // @todo make this a subscription
+  const entityStore = entityStorePluginStateFromTransaction(
+    blockView.view.state.tr,
+    blockView.view.state,
+  ).store;
```

Comment:
You can just use `entityStorePluginState` here as the transaction you're passing is new (`tr` is a getter which creates a transaction), so it won't have state associated with it, so its just calling `entityStorePluginState` anyway.

Reply:
Makes sense. Updated here => [c4ed9f6](https://github.com/hashintel/hash/pull/480/commits/c4ed9f66793158070235f7d5e9545415a8019f34)